reddit_liveupdate/templates/liveupdateeventapp.html

```diff
@@ -8,6 +8,11 @@
 
 <header id="liveupdate-header">
   <h1 id="liveupdate-title">${thing.event.title}</h1>
+  % if thing.event.description:
+  <div id="liveupdate-description">
```

Comment:
I think this element needs `class="md-container"` so that it properly gets the markdown stylin'.  Otherwise the text is rather tiny.